src/rpc/methods/state.rs

```diff
@@ -572,6 +575,74 @@ pub async fn state_miner_available_balance<DB: Blockstore + Send + Sync + 'stati
     Ok(LotusJson(vested + available))
 }
 
+pub async fn state_miner_initial_pledge_collateral<DB: Blockstore + Send + Sync + 'static>(
```

Comment:
Let's use the new RPC framework instead!

Reply:
Sure, that was my plan, once api compare tests passed.